docs/src/pages/[platform]/connected-components/storage/fileuploader/react.mdx

```diff
@@ -24,6 +24,12 @@ import {
   UploadActionsExample,
 } from './examples'
 
+<Alert variation="info" heading="Storage Manager is renamed">
+  StorageManager component has been renamed `FileUploader` as of `aws-amplify/ui-react-storage` version `3.3.0`. Please upgrade to the new import path, as the `StorageManager` will be removed in a future major version.
```

Comment:
```suggestion
  StorageManager component has been renamed `FileUploader` as of `@aws-amplify/ui-react-storage` version `3.3.0`. Please upgrade to the new import path, as the `StorageManager` will be removed in a future major version.
```

Reply:
```suggestion
      Beginning with @aws-amplify/ui-react-storage@v3.3.0, the StorageManager
      component has been renamed to FileUploader. To migrate, upgrade to the
      latest version of @aws-amplify/ui-react-storage and replace StorageManager
      related imports with their corresponding FileUploader equivalents.
      
      StorageManager will continue to receive official support until the next
      major version release of @aws-amplify/ui-react-storage, at which point
      it will be replaced by FileUploader.
```

Reply:
Let's go with Caleb's more detailed one

Reply:
Agreed. I just made a slight change of `_@aws-amplify/ui-react-storage@v3.3.0_` to `_@aws-amplify/ui-react-storage_ version 3.3.0`, as the original created unwanted links and the new pattern is in line with existing references on the page.